src/com/epam/task/string/Task8.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.task.string;
+
+import static com.epam.task.util.InputUtils.inputString;
+
+public class Task8 {
+
+    public static void main(String[] args) {
+        String string = inputString("Enter string: ");
+        printWordWithMaxLength(string);
+    }
+
+    public static int findMaxLengthOfWord(String string) {
+        String[] strings = string.split("\\s+");
```

Comment:
Lets move \\s+ to the constants

Reply:
Also, I think the better way to implement this task is to sort this array by the length of the words and then just get the last element